app/assets/images/payment_icons/dpo-vodaphonempesa.svg

```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img"
```

Comment:
The border radius doesn't match other payment icons.

[Video demonstration](https://screenshot.click/22-32-hluvs-aisg8.mp4)